.github/actions/setup/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Setup F´'
+description: 'Setup the F´ repository and tools'
+inputs:
+  location:
+    required: true
+    default: $GITHUB_WORKSPACE
+runs:
+  using: "composite"
+  steps:
+  - run: git fetch --tags
+    shell: bash
+  - run: pip3 install setuptools_scm urllib3
```

Comment:
Would it be possible to make these dev dependencies of the setup.py file? If we have dependencies we should make sure they're properly documented through a requirements file/setup.py/install instructions rather than hidden in a github action.

Reply:
No, these dependencies are of the setup.py file, and must exist before pip is run. The fix is to not use them.  That will is already fixed but will come in when we publish fpp version 1.0.2.

Reply:
Does this need to be documented anywhere for users of this metapackage?

Reply:
I don't think so.  I am not officially releasing this package until v3.0.1 and v1.0.2.  That is when I'll add it to the install instructions and it should work without that preinstall.

Until the release, I need CI to pass.